pages/common/hadolint.md

```diff
@@ -0,0 +1,16 @@
+# hadolint
+
+> Lints Dockerfiles to help you build [best practice](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices) Docker images.
+> More information: https://github.com/hadolint/hadolint.
+
+- Lint a Dockerfile, printing any output to STDOUT:
+
+`hadolint {{Dockerfile}}`
+
+- Lint a Dockerfile ignoring specific rules and outputting JSON:
+
+`hadolint --ignore {{DL3006}} --format json {{Dockerfile}}`
```

Comment:
You need to say how to add a 2nd rule, could be a coma or space-separated list of rules, but for this command I saw this is the way:
```suggestion
`hadolint --ignore {{DL3006}} --ignore {{DL3008}} --format json {{path/to/Dockerfile}}`
```

Reply:
Tested and verified only way to pass through multiple rules is to use the `--include` flag multiple times